### DISCUSSION TITLE
Is there a way to stop the decodeAudioData-function? (With the callback-system, not the Promise)

### DISCUSSION THREAD

**Discussion by MaxJoLi** [WebAudio/web-audio-api#2284] [Q&A]
More out of curiosity / global understanding / for future reference than technical necessity at the moment: is it possible to stop the decodeAudioData-function when using it via the success and error-callback system?

**Comment by rtoy** [comment]
I believe the answer is "no".  There's no mechanism defined to do that.

In general, I think we're moving in the direction of using WebCodecs instead of decodeAudioData.
